rpc/json2/server_test.go

```diff
@@ -19,11 +19,12 @@ package json2
 import (
 	"bytes"
 	"errors"
-	"github.com/ChainSafe/gossamer/rpc"
 	"log"
 	"net/http"
 	"strings"
 	"testing"
+
+	"github.com/ChainSafe/gossamer/rpc"
```

Comment:
Same as above

Reply:
Simply ran the linting command with goimports enabled and it fixed accordingly....I have no preference